### PR TITLE
Add labwc workaround

### PR DESCRIPTION
**Pull request by nathan-contino**
As discussed in https://github.com/raspberrypi/documentation/issues/3353

### CODE REVIEW

File: documentation/asciidoc/accessories/display/display_intro.adoc [hunk 1]
```diff
@@ -132,6 +132,27 @@ Then, disable automatic display detection by removing the following line from `c
...
+----
+
+Then, use the Raspberry Pi Configuration CLI to switch to the `labwc` window compositor. Run the following command to open `raspi-config`:
+
+[source,console]
+----
+$ sudo raspi-config
+----
+
+Go to **Advanced Options**. Select **Wayland**, then select **labwc**. Exit `raspi-config` and reboot with `sudo reboot`. Switching to `labwc` may reset some configuration, including keyboard layout.
```
Comment (steveiliop56) [inline documentation/asciidoc/accessories/display/display_intro.adoc:153]: You can also add that it will likely reset rotation settings.

Comment (nathan-contino) [reply documentation/asciidoc/accessories/display/display_intro.adoc:153]:
```suggestion
Go to **Advanced Options**. Select **Wayland**, then select **labwc**. Exit `raspi-config` and reboot with `sudo reboot`. Switching to `labwc` may reset some configuration, including keyboard layout and rotation settings.
```

File: documentation/asciidoc/accessories/display/display_intro.adoc [hunk 2]
```diff
@@ -132,6 +132,27 @@ Then, disable automatic display detection by removing the following line from `c
 display_auto_detect=1
 ----
 
+[NOTE]
+====
+In Raspberry Pi OS _Bookworm_, a bug in https://github.com/WayfireWM/wayfire[Wayfire] currently prevents cursor movement rotation. There is no workaround for this bug in Wayfire. Instead, you can use https://github.com/labwc/labwc[`labwc`] (currently in beta).
```
Comment (6by9) [inline documentation/asciidoc/accessories/display/display_intro.adoc:137]: It incorrectly rotates the cursor when using WayVNC. Locally it is fine.

AIUI we aren't be looking for a fix in Wayfire.
Labwc is the route we are taking forward as it should be work reasonably on the earlier Pis, and hope to adopt it very soon.

Comment (steveiliop56) [reply documentation/asciidoc/accessories/display/display_intro.adoc:137]: Yeah I agree but what I am saying is that the problem of the inverted mouse in wayvnc is a wayfire problem of not assigning something correctly. I am just recommending labwc to people that just want to make wayvnc with rotation work.

Comment (nathan-contino) [reply documentation/asciidoc/accessories/display/display_intro.adoc:137]: Fixing this in a separate PR. Thanks!

**Pull request merged**